lib/greeve/eve/alliance_list.rb

```diff
@@ -0,0 +1,34 @@
+require_relative "../base_item"
+
+module Greeve
+  module Eve
+    # Returns active Alliances in New Eden and their member Corporations.
```

Comment:
`Alliances` and `Corporations` can be lowercase. I know this is straight from the docs, but their grammar is wrong.